Bio/Seq.py

```diff
@@ -62,7 +63,7 @@ def _maketrans(complement_mapping):
 
 
 class Seq(object):
-    """A read-only sequence object (essentially a string with an alphabet).
+    """Create a read-only sequence object (essentially a string with an alphabet).
```

Comment:
The ``__init__`` could be said to create the object, but that does not make sense for the class docstring. You could just remove the ``A`` and use ``Read-only sequence object (essentially a string with an alphabet).``